### PR TITLE
Fix py testcases and related issues in the interface

### PR DESCRIPTION
**Pull request by RudolfWeeber**

### CODE REVIEW

File: testsuite/python/particle.py [hunk 1]
```diff
@@ -138,8 +138,17 @@ def func(self):
     if "VIRTUAL_SITES" in espressomd.features():
         test_virtual = generateTestForScalarProperty("virtual", 1)
     if "VIRTUAL_SITES_RELATIVE" in espressomd.features():
-        test_zz_vs_relative = generateTestForScalarProperty(
-            "vs_relative", ((0, 5.0)))
+        def test_zz_vs_relative(self):
+            self.es.part.add(id=0,pos=(0,0,0))
+            self.es.part.add(id=1,pos=(0,0,0))
+	    self.es.part[1].vs_relative = (0, 5.0,(0.5,-0.5,-0.5,-0.5))
```
Comment (KaiSzuttor) [inline testsuite/python/particle.py]: identation is a broken here

File: testsuite/python/magnetostaticInteractions.py [hunk 2]
```diff
@@ -50,10 +50,8 @@ def paramsMatch(self, inParams, outParams):
 
     def setUp(self):
         self.system.box_l = 10, 10, 10
-        self.system.part[0].pos = 0, 0, 0
-        self.system.part[1].pos = 0.1, 0.1, 0.1
-        self.system.part[0].dip = 1.3, 2.1, -6
-        self.system.part[1].dip = 4.3, 4.1, 7.5
+        self.system.part.add(id=0,pos = (0.1, 0.1, 0.1),dip = (1.3, 2.1, -6))
```
Comment (KaiSzuttor) [inline testsuite/python/magnetostaticInteractions.py]: According to PEP8 its nicer to have spaces after a comma. Just for readability.

**Comment by RudolfWeeber** [comment]
Sorted out the vs_relative test, removed a bunch of debug output and autopep8

**Comment by KaiSzuttor** [comment]
This can be merged now.

**Pull request merged**